docs/8.0-ServiceQuotas.md

```
@@ -103,20 +104,38 @@ status:
+
 ## How to update service quotas for AccountPool owned accounts?
 Begin by updating the AAO ConfigMap and then restarting the `aws-account-operator` pod. Upon the update, the `accountpool_validation_controller.go` will parse the modified AAO ConfigMap using the `GetServiceQuotasFromAccountPool` function: [https://github.com/openshift/aws-account-operator/blob/4af362e874f99454c7b69166588c4536728bbfc5/pkg/utils/utils.go#L116-L165]
 
-Subsequently, within the `accountpool_validation_controller.go`, all accounts associated with an accountpool will be iterated over. It will inspect the `account.Spec.RegionalServiceQuotas` for any discrepancies compared to the ConfigMap, updating the spec accordingly: [https://github.com/openshift/aws-account-operator/blob/91d7d76294ebd2d9e6da512b0ed2fd7d7cf759b2/controllers/validation/accountpool_validation_controller.go#L126-L182]
+Subsequently, within the `accountpool_validation_controller.go`, all accounts associated with an accountpool will be iterated over. It will inspect both `account.Spec.RegionalServiceQuotas` and `account.Spec.GlobalServiceQuotas` for any discrepancies compared to the ConfigMap, updating the spec accordingly. If either regional or global quotas have drifted, both are reconciled and the account status quotas are reset.
 
-The updated account spec will then trigger the `account_validation_controller.go`, where the handling of service quota requests will occur [link to code](https://github.com/openshift/aws-account-operator/blob/91d7d76294ebd2d9e6da512b0ed2fd7d7cf759b2/controllers/validation/account_validation_controller.go#L457-L530)
+The updated account spec will then trigger the `account_validation_controller.go`, where the handling of both regional and global service quota requests will occur.
 
 ## How to add support for additional service quotas?
 #### Step 1: Define Service and Quota Codes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)